Signal/src/call/SignalCall.swift

```diff
@@ -41,6 +41,16 @@ protocol CallDelegate: class {
     var state: CallState {
         didSet {
             Logger.debug("\(TAG) state changed: \(oldValue) -> \(state)")
+
+            // Update connectedDate
+            if state == .connected {
+                if connectedDate == nil {
+                    connectedDate = NSDate()
+                }
+            } else {
```

Comment:
>I think we should continue to display the time (frozen) after the other party hangs up.

I'm using the same UILabel to display the "call status" (Dialing, Call Ended) and the "call duration", just like the iOS call screen.   The "call duration" is shown while the call is connected. When the call ends, the call status changes to something like "Call Ended," again, just like the iOS call screen.  I like how the iOS call screen works - what's the argument for deviating?